cmake/ExplicitInstantiationSupport.cmake

```diff
@@ -0,0 +1,96 @@
+include(Join)
```

Comment:
Add a comment that this file is equivalent to Ifpack2's. This way later we can remember to sync if something changes.